src/data/links.yaml

```diff
@@ -1254,6 +1257,7 @@ links:
     url: https://cariuma.com/
     categories:
       - sustainable-lifestyle
```

Comment:
We can remove things from sustainable lifestyle as we add them to clothing, that category became a bit of a catch all.

Reply:
Okay. I was not sure what the protocol was here since I saw some other links with multiple categories.